### PR TITLE
Improve abstract and introduction

### LINKED ISSUE #315: Security and Privacy Guidelines not listed as building block in the introduction
Please add the text and link to the bullet list.

### PR DESCRIPTION
**Pull request by mkovatsc**
This fixes #315 and reduces the redundancy between abstract and introduction. Several outdated phrases were updated.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -217,50 +217,47 @@
...
-            <li>the <em>Web of Things (WoT)
-                    Scripting API</em> [[?wot-scripting-api]].
-            </li>
-        </ul>
-
+	    
+        <p>This document is focused on the scope of W3C WoT standardization,
+	    which is broken down into so-called building blocks.
+	    It introduces the four intial WoT building blocks,
+            which are described in detail in separate documents,
```
Comment (mlagally) [inline index.html]: please use specifications, or specification documents instead of just "documents"

File: index.html [hunk 2]
```diff
@@ -279,53 +276,55 @@
...
-            specifications and defines the basics such as terminology
+        <p>This document serves as an umbrella for the W3C WoT
+            documents and defines the basics such as terminology
             and the underlying abstract architecture of the W3C Web of
             Things. In particular, the purpose of this document is to
             provide:</p>
         <ul>
-            <li>a set of use cases that lead to the W3C WoT
-                Architecture in chapter <a href="#sec-use-cases"></a>,
+            <li>a set of use casesin <a href="#sec-use-cases"></a>
```
Comment (mlagally) [inline index.html]: typo

File: index.html [hunk 3]
```diff
@@ -279,53 +276,55 @@
...
-                software stacks and hardware components in chapter <a
-                href="#sec-servient-implementation"></a>,
+                software stacks and hardware components in
+                <a href="#sec-servient-implementation"></a>,
             </li>
-            <li>deployment scenarios in chapter <a
+            <li>deployment scenarios in <a
                 href="#sec-deployment-scenario"></a>,
             </li>
             <li>and security considerations to be aware of when
```
Comment (mlagally) [inline index.html]: to be aware -> to be applied ?

File: index.html (hunk 1 above)
Comment (mkovatsc) [reply index.html]: Are the WG Notes specifications?

File: index.html (hunk 3 above)
Comment (mkovatsc) [reply index.html]: To my understanding, the ones to be applied are in the Sec Guidelines document.
"Security Considerations" sections are to make readers of the spec aware.
No?

File: index.html (hunk 2 above)
Comment (mkovatsc) [reply index.html]: Done

File: index.html (hunk 1 above)
Comment (mlagally) [reply index.html]: sure.

File: index.html (hunk 3 above)
Comment (mlagally) [reply index.html]: I guess it is more or less synonymous. We can keep it.

**Comment by mlagally** [comment]
Ready to merge.

**Comment by mkovatsc** [comment]
Changing "document" to "specification" where talking about our W3C documents.
Will merge when done.

**Pull request merged**